packages/camera/camera/README.md

```diff
@@ -100,6 +100,7 @@ Here is a small example flutter app displaying a full screen camera preview.
 <?code-excerpt "readme_full_example.dart (FullAppExample)"?>
 ```dart
 import 'package:camera/camera.dart';
+import 'package:camera_platform_interface/camera_platform_interface.dart';
```

Comment:
We don't want to require a user to import the platform interface package. You should export MediaSettings in lib/camera.dart.

Reply:
Exported. Removed.

Reply:
This is still here.